app/models/repo/challenge.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Repo
+  class Challenge < ApplicationRecord
+    extend FriendlyId
+    friendly_id :title, use: :slugged
+
+    DRAFT = 'draft'
```

Comment:
```suggestion
    STATUSES = [
        DRAFT = 'draft',
        MODERATION = 'moderation',
        ...
    ].freeze
```

Reply:
Why do you think this approach is better?